### PR TITLE
#155965773 Create the GitHubUsersAdapter

### PR DESCRIPTION
**Pull request by wycliffkas**
#### What does this PR do?
- creates a GitHubUsersAdapter file
#### Description of Task to be completed?
- add GitHubUsersAdapter file to the adapter package
#### What are the relevant pivotal tracker stories?
[#155965773](https://www.pivotaltracker.com/story/show/155965773)

### CODE REVIEW

**Comment by huxaiphaer** [comment]
Squash the commits.

File: app/src/main/java/com/example/javadevelopersnairobi/javadevelopersnairobi/model/Data.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import java.util.List;
+
+public class Data {
+//	@SerializedName("GithubUsersResponse")
+//	private GithubUsersResponse githubUsersResponse;
+//
+//	public GithubUsersResponse getGithubUsersResponse(){
+//		return githubUsersResponse;
+//	}
```
Comment (huxaiphaer) [inline app/src/main/java/com/example/javadevelopersnairobi/javadevelopersnairobi/model/Data.java]: Remove these commented lines.

**Review by huxaiphaer — COMMENTED** [review]
Work on the feedback.

File: app/src/main/java/com/example/javadevelopersnairobi/javadevelopersnairobi/model/Data.java (hunk 1 above)
Comment (wycliffkas) [reply app/src/main/java/com/example/javadevelopersnairobi/javadevelopersnairobi/model/Data.java]: Thank you for the heads ups, am going to remove them.

**Review by huxaiphaer — APPROVED** [review]
LGTM

**Pull request merged**